lib/foreman_ansible/register.rb

```diff
@@ -98,6 +98,10 @@
 
   register_global_js_file 'global'
 
+  extend_graphql_type :type => ::Types::Host do
+    field :all_ansible_roles, ::Types::AnsibleRole.connection_type, :null => true
```

Comment:
This seems to be causing the tests to fail with
```
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "ansible_roles" does not exist
LINE 8:  WHERE a.attrelid = '"ansible_roles"'::regclass
```
I guess it runs before the db is migrated so the table doesn't yet exit?